### PR TITLE
[Merged by Bors] - feat(Measure/Typeclasses): prove `≪` both ways

### PR DESCRIPTION
**Pull request by urkud**
- Rename `exists_absolutelyContinuous_isFiniteMeasure` to `exists_isFiniteMeasure_absolutelyContinuous` to match the statement.
- Prove `ν ≪ μ`, not only `μ ≪ ν`.
- Turn the old lemma into a deprecated alias.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/Typeclasses.lean [hunk 1]
```diff
@@ -536,17 +536,22 @@ instance [SFinite μ] (s : Set α) : SFinite (μ.restrict s) :=
...
-    have : ∀ n, (sFiniteSeq μ n) s = 0 := by simpa [hsm, (hc₀ _).ne'] using hs
-    rw [← sum_sFiniteSeq μ, sum_apply _ hsm]
-    simp [this]
+  refine ⟨.sum fun n ↦ c n • sFiniteSeq μ n, ⟨by simpa [mul_comm] using hc⟩, fun s ↦ ?_⟩
+  conv_rhs => rw [← sum_sFiniteSeq μ, sum_apply_of_countable]
+  simp [(hc₀ _).ne']
+
+variable (μ) in
+/-- An s-finite measure is absolutely continuous with respect to some finite measure. -/
+@[deprecated exists_isFiniteMeasure_null_iff (since := "2024-08-25")]
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Measure/Typeclasses.lean:550]: I don't think we should deprecate this one: often, it is the one you need for the applications.

File: Mathlib/MeasureTheory/Measure/Typeclasses.lean [hunk 2]
```diff
@@ -536,17 +536,22 @@ instance [SFinite μ] (s : Set α) : SFinite (μ.restrict s) :=
     by rw [← restrict_sum_of_countable, sum_sFiniteSeq]⟩
 
 variable (μ) in
-/-- An s-finite measure is absolutely continuous with respect to some finite measure. -/
-theorem exists_absolutelyContinuous_isFiniteMeasure [SFinite μ] :
-    ∃ ν : Measure α, IsFiniteMeasure ν ∧ μ ≪ ν := by
+/-- For An s-finite measure, there exists a finite measure with the same null sets. -/
+theorem exists_isFiniteMeasure_null_iff [SFinite μ] :
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Measure/Typeclasses.lean:540]: The name is a little bit misleading (I expect that a lemma ending with `iff` is an equivalence). Maybe `exists_isFiniteMeasure_null_iff_null` to lift the ambiguity?

File: Mathlib/MeasureTheory/Measure/Typeclasses.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/MeasureTheory/Measure/Typeclasses.lean]: What do you think about the new version?

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Pull request closed without merging**